extensions/wikia/ARecoveryEngine/ResourceLoaders/ResourceLoaderAdEngineSourcePointBase.php

```diff
@@ -23,7 +28,7 @@ protected function getFallbackDataWhenRequestFails() {
 		$scripts = [
 			( new ResourceLoaderScript() )
 				->setTypeLocal()
-				->setValue($this->fallbackScriptUrl)
+				->setValue( static::FALLBACK_SCRIPT_URL )
 		];
 		$data = [
 			'script' => $this->generateData( $scripts ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "script" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)